homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -74,6 +75,8 @@
 # implemented for top/down shades, but also works fine with normal shades
 CLOSED_POSITION = (0.75 / 100) * (MAX_POSITION - MIN_POSITION)
 
+SCAN_INTERVAL = timedelta(minutes=5)
```

Comment:
We might be able to lower this but I was worried that we wouldn't be able to update all the shades in time since it took about 5-6s per shade.

The current setting can roughly handle 50-60 hard wired shades. I would be surprised if the hub itself can handle that many given how flakey they are

Reply:
I personally was thinking a 15 minute sync - I think 5 will be plenty if not overkill - I would consider making to 10 ?

people don’t move their shades that often

and the hubs really don’t enjoy being spammed for an update

I know they on occasion return an additional property lf timeout=true instead of actually returning new data iif updating is done too often

Reply:
10 seems safer so lets do that.  That means it should still work with 100 shades which seems like its probably beyond what the hub can support anyways

Reply:
> people don’t move their shades that often

I've said that before as well and been told that was not a good assumption.  I pointed out that it probably wasn't a good idea to move them too frequently as the motors are not rated for unlimited operations.

I expect someone is going to ask for it to be adjusted to a faster interval within days of the this being in a release.  The solution for that will be to use the automation in the docs.